network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/ConnectablePositionImpl.java

```diff
@@ -68,11 +98,108 @@ public Direction getDirection() {
+            branch.updateResource(res -> {
+                if (getSide() == ThreeSides.ONE) {
+                    res.getAttributes().getPosition1().setOrder(order);
+                } else if (getSide() == ThreeSides.TWO) {
+                    res.getAttributes().getPosition2().setOrder(order);
+                }
+            });
+        }
+
+        private void updateTransformerOrderResource(ThreeWindingsTransformerImpl windingsTransformer, int order) {
+            windingsTransformer.updateResource(res -> {
+                if (getSide() == ThreeSides.ONE) {
+                    res.getAttributes().getPosition1().setOrder(order);
+                } else if (getSide() == ThreeSides.TWO) {
+                    res.getAttributes().getPosition2().setOrder(order);
+                } else if (getSide() == ThreeSides.THREE) {
+                    res.getAttributes().getPosition3().setOrder(order);
+                }
+            });
+        }
+
+        private void updateInjectionDirectionResource(AbstractInjectionImpl<?, ?> injection, Direction direction) {
+            injection.updateResource(res -> res.getAttributes().getPosition()
+                    .setDirection(Direction.valueOf(Objects.requireNonNull(direction).name())));
+        }
+
+        private void updateBranchDirectionResource(AbstractBranchImpl<?, ?> branch, Direction direction) {
+            branch.updateResource(res -> {
+                if (getSide() == ThreeSides.ONE) {
+                    res.getAttributes().getPosition1().setDirection(Objects.requireNonNull(direction));
+                } else if (getSide() == ThreeSides.TWO) {
+                    res.getAttributes().getPosition2().setDirection(Objects.requireNonNull(direction));
+                }
+            });
+        }
+
+        private void updateTransformerDirectionResource(ThreeWindingsTransformerImpl windingsTransformer, Direction direction) {
+            windingsTransformer.updateResource(res -> {
+                if (getSide() == ThreeSides.ONE) {
+                    res.getAttributes().getPosition1().setDirection(Objects.requireNonNull(direction));
+                } else if (getSide() == ThreeSides.TWO) {
+                    res.getAttributes().getPosition2().setDirection(Objects.requireNonNull(direction));
+                } else if (getSide() == ThreeSides.THREE) {
+                    res.getAttributes().getPosition3().setDirection(Objects.requireNonNull(direction));
+                }
+            });
+        }
+
+        private void notifyUpdate(AbstractConnectableImpl connectable, String attribute, Object oldValue, Object newValue) {
+            String variantId = connectable.getNetwork().getVariantManager().getWorkingVariantId();
+            connectable.getNetwork().getIndex().notifyUpdate(getExtendable(), attribute, variantId, oldValue, newValue);
+        }
     }
 
+    private ThreeSides side;
```

Comment:
You use the side only in FedderImpl, so why not storing it in FeederImpl class ?